tests/test_preExecInit.py

```diff
@@ -121,6 +121,22 @@ def test_savePackageVersions_twice(self):
                         with self.assertRaises(Exception):
                             preExecInit.savePackageVersions(qgraph)
 
+    def test_datasetTypeNames(self):
```

Comment:
move to pipe_base?

Reply:
You convinced me :slightly_smiling_face: I moved that PreExecInit method and this test to pipe_base (PipelineDatasetTypes class). Next logical step would be to move the whole PreExecInit to pipe_base too, but this should wait until big refactoring.